package.json

```diff
@@ -5,30 +5,21 @@
     "license": "CC-BY-3.0",
     "repository": "https://github.com/fluid-project/infusion-docs",
     "scripts": {
-        "docpad": "node node_modules/docpad/bin/docpad run --env static --out out",
-        "deploy": "node node_modules/docpad/bin/docpad deploy-ghpages --env static",
-        "pretest": "node node_modules/docpad/bin/docpad generate --env static --out out",
+        "11ty": "mkdir -p out && eleventy --serve",
+        "build": "mkdir -p out && eleventy",
         "test": "npm run test:node && npm run test:browser",
         "test:node": "node tests/all-node-tests.js",
         "test:browser": "node node_modules/testem/testem.js ci --file tests/testem.js"
     },
     "dependencies": {
-        "docpad": "6.79.4",
-        "docpad-plugin-ghpages": "2.6.1",
-        "docpad-plugin-handlebars": "2.4.0",
-        "docpad-plugin-highlightjs": "2.5.0",
-        "docpad-plugin-markit": "2.4.0",
+        "@11ty/eleventy": "0.11.0",
         "foundation-sites": "5.5.2",
-        "fs-extra": "4.0.2",
         "gpii-binder": "1.0.5",
         "gpii-location-bar-relay": "1.0.4",
-        "infusion": "3.0.0-dev.20190328T144119Z.ec44dbfab",
+        "highlight.js": "10.0.3",
+        "infusion": "3.0.0-dev.20200326T173810Z.24ddb2718",
```

Comment:
infusion is just being copied through makethroughCopy of 11ty. So we can put it under dev dependencies too.

Reply:
Are you sure about this? Aren't devDeps not installed in the production server? So 11ty won't have a folder to make a passthrough copy and infusion is used extensively. Are you sure all of those usages are dev-specific? If yes, then I'll add them to devdeps. Also, it was originally in the main deps.
It may be that I have a lesser understanding of this.

Reply:
I tested it on local and it seems to be working.

Reply:
We had a similar discussion and @jhung suggested this.

Reply:
Okay I'll shift infusion to devdeps. Thanks for checking.
Also, are there other packages that are required to be shifted to devdeps. Since I mistook infusion, I may be wrong for others too.

Reply:
> I tested it on local and it seems to be working.

Are you sure you ran `npm install --production` because running the same in my local didn't get infusion.

Reply:
Floe and fluid have been deployed using 11ty and both have infusion in devdependencies .sites are working fine